packages/components/src/animate/styles/animate-styles.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import * as animations from './animate-keyframe-styles';
+import { reduceMotion } from '../../utils';
+import { getRTLSafeTranslateX, getRTLSafeXAxis } from './utils';
+
+export const AppearingWrapper = styled.div`
+	animation: ${ animations.appear } 0.1s cubic-bezier( 0, 0, 0.2, 1 ) 0s;
+	animation-fill-mode: forwards;
+
+	${ reduceMotion( 'animation' ) }
+    
+	transform-origin: ${ ( p ) => p.yAxis } ${ getRTLSafeXAxis };
```

Comment:
@saramarcondes Haii! It looks like you have the RTL figured out :).

My suggestion here would be to abstract the `transform-origin` logic into another function.
My thinking for that is because having all inline is tricky to read (at least for me).

Abstracting it will allow us to write something like this:

```
${ appearTransformOrigin };
```
(or something similar)

And the prop interpolation + RTL would be handled in that :)

The same could apply for the `SliderWrapper`.

P.S. I found that outputting the styles in the abstracted function works best using the `css` function from emotion, example:

https://github.com/WordPress/gutenberg/blob/master/packages/components/src/input-control/styles/input-control-styles.js#L148

It seems to play nicer with editors, prettier, and linters.

Note: It could be used with string interpolation or as a function `css({ ...})`.

<3